stripe/src/test/java/com/stripe/android/googlepay/StripeGooglePayViewModelTest.kt

```diff
@@ -156,12 +167,47 @@ class StripeGooglePayViewModelTest {
         )
     }
 
+    @Test
+    fun `createPaymentMethod() with stripeAccount should include stripeAccount in request`() {
+        val stripeAccout = "account_id"
+        var requestOptions: ApiRequest.Options? = null
+        val stripeRepository = object : AbsFakeStripeRepository() {
+            override suspend fun createPaymentMethod(
+                paymentMethodCreateParams: PaymentMethodCreateParams,
+                options: ApiRequest.Options
+            ): PaymentMethod? {
+                requestOptions = options
+                return null
+            }
+        }
+        val viewModel = StripeGooglePayViewModel(
+            ApplicationProvider.getApplicationContext(),
+            ApiKeyFixtures.FAKE_PUBLISHABLE_KEY,
+            stripeAccout,
+            ARGS,
+            stripeRepository,
+            "App Name",
+            testDispatcher
+        )
+
+        val params = PaymentMethodCreateParams.createFromGooglePay(
+            GOOGLE_PAY_RESULT_WITH_NO_BILLING_ADDRESS
+        )
+
+        viewModel.createPaymentMethod(params).observeForever {
+            assertThat(requestOptions?.stripeAccount).isEqualTo(stripeAccout)
+        }
+
+        countingTaskExecutorRule.drainTasks(3, TimeUnit.SECONDS)
```

Comment:
Why is this necessary?

Reply:
It's actually not. I was testing in a different way that was more complex and needed to wait for the workContext to finish.